tasks/disk_drive.yml

```diff
@@ -3,7 +3,9 @@
 
 #following expects that your system has only same type disks on thats why this incredible stupid hack
 - name: Identifying firmware RPM for disk drives
-  shell: "yum search `hpssacli ctrl all show config detail | grep 'Model:' |grep -E '(ATA|HP)' | sort -u| awk '{print $NF}'` | grep -i hp-firmware | awk '{print $1}' | grep `uname  -m`"
+  shell: "yum search `{{ ss_utility }} ctrl all show config detail | grep 'Model:' |grep -E '(ATA|HP)' | sort -u| awk '{print $NF}'` | grep -i firmware-hdd | awk '{print $1}' | grep `uname  -m`"
```

Comment:
The pattern matching now does: 

```
 grep -i firmware-hdd 
```

instead of 

```
grep -i hp-firmware 
```

Why was this changed?

Reply:
Hello @FLiPp3r90 ,

which yum repo are you using?

https://downloads.linux.hpe.com/SDR/repo/spp/RHEL/7/x86_64/current/ has lots of RPMs that start with "hp-firmware". 
Anyway, I like the change to use firmware-hdd in the grep because it matches on newer firwmares and seems we have probably not updated our HDD firmwares in a while! So thank you again for this PR!

As an example (I just took a random HDD) the above repo has these:

```
hp-firmware-hdd-fa46c607d6-HPD1-1.1.x86_64.rpm                        2016-08-30 04:32  4.8M  
firmware-hdd-fa46c607d6-HPD2-1.1.x86_64.rpm                           2017-05-02 22:39  5.0M  
firmware-hdd-fa46c607d6-HPD3-1.1.x86_64.rpm                           2018-04-12 13:37  9.4M  
```

We have some new hardware coming in so we should be able to test this without breaking anything important in the next week or two.

Reply:
I've referred to the HP repo https://downloads.linux.hpe.com/SDR/repo/spp-gen9/2018.11.0/packages/. 
In the RHEL packages are always named like hp-firmware-hdd.*